### PR TITLE
Move kernel scripts parameters to kworkflow.config

### LINKED ISSUE #260: Move kernel script parameters to kworkflow.config
kw already have a wrapper to get_maintaners and checkpack scripts. However, the parameters used by these scripts are hardcoded; let's move them to the kworkflow.config and allow users to use the parameter that they want.

### PR DESCRIPTION
**Pull request by kwy95**
This PR moves the parameters used by the `checkpatch_wrapper.sh` and `get_maintainer_wrapper.sh` scripts to the *kworkflow.config* file.

Closes #260

### CODE REVIEW

File: src/checkpatch_wrapper.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ function execute_checkpatch()
   # output. It could be nice if we can add another option just for this sort
   # of check.
 
-  local options='--no-tree --color=always --strict'
+  local options="${configurations[checkpatch_opts]}"
```
Comment (rodrigosiqueira) [inline src/checkpatch_wrapper.sh:25]: What happens if `options` is empty? Do we have a test to check this behavior?

**Review by rodrigosiqueira — CHANGES_REQUESTED** [review]

**Comment by kwy95** [comment]
@rodrigosiqueira
Will update soon.

File: src/checkpatch_wrapper.sh (hunk 1 above)
Comment (kwy95) [reply src/checkpatch_wrapper.sh:25]: I think having no options is a valid usage. If the user clears this option we shouldn't assume any options.

**Comment by rodrigosiqueira** [comment]
@kwy95 did you update this PR?

**Comment by kwy95** [comment]
@rodrigosiqueira The only concern you presented was about the script parameters being empty, but I believe that to be a valid usage. What do you think?

**Comment by rodrigosiqueira** [comment]
Hi @kwy95 ,
This PR lgtm, could you just rebase it with the unstable branch?

**Comment by rodrigosiqueira** [comment]
Merged, thanks

**Pull request closed without merging**